### PR TITLE
Closes 1 - Create initial index.html markup

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
```
Comment (tkarpli) [inline index.html]: Please, add new line

Comment (IvannaChyrva) [inline index.html]: Please add new line

Comment (NazarHarashchak) [inline index.html]: Please add a line

Comment (AndriiBuchman) [inline index.html]: Please add new line

**Review by NazarHarashchak — CHANGES_REQUESTED** [review]
Please add a line

**Review by IvannaChyrva — COMMENTED** [review]
...

**Review by stasglazkov1995 — CHANGES_REQUESTED** [review]
Empty body

**Review by tkarpli — CHANGES_REQUESTED** [review]
See comments

**Review by AndriiBuchman — CHANGES_REQUESTED** [review]
see comment

**Review by AndriiBuchman — APPROVED** [review]

**Review by tkarpli — APPROVED** [review]

**Review by NazarHarashchak — APPROVED** [review]

**Review by stasglazkov1995 — APPROVED** [review]

**Review by IvannaChyrva — APPROVED** [review]

**Pull request merged**